react-native/codepush-hockeyapp/appcenter-post-build.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Example: Install App Center CLI
+npm install -g appcenter-cli
```

Comment:
App Center CLI should be [already available](https://docs.microsoft.com/en-us/appcenter/build/software#tools) - is there a specific reason you're prompting an install?

Reply:
Ohh thanks for the feedback, let me remove this line